src/microbe_stage/Microbe.cs

```diff
@@ -584,6 +609,25 @@ public void ApplyMembraneWigglyness()
             Membrane.Type.MovementWigglyness) * 0.2f;
     }
 
+    /// <summary>
+    ///   Gets the actually hit microbe (potentially in a colony)
+    /// </summary>
+    /// <param name="bodyShape">The shape that was hit</param>
+    /// <returns>The actual microbe that was hit</returns>
+    public Microbe GetActualHitMicrobe(int bodyShape)
```

Comment:
I'm not very happy with this naming. I suggest something like `GetMicrobeFromShape` and state in the summary that this is needed because all colony shapes are owned by the master